views/changeEmailSuccess.njk

```diff
@@ -18,8 +18,9 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
-        {% set message = "A verification email has been sent to " + email  +
-          (".<br>The new address will now have to be used to sign in." if usernameChanged) %}
+        {% set message %}
+          A verification email has been sent to {{ email }}{{ ".<br>The new address will now have to be used to sign in." if usernameChanged }}
+        {% endset %}
```

Comment:
Unlikely to want to XSS yourself by somehow changing email to include a XSS attack, but while I'm at it...